terraform-plans/templates/github/snap_tics.yaml.tftpl

```diff
@@ -23,25 +23,26 @@ jobs:
         run: |
           sudo apt update
           python -m pip install --upgrade pip
-          # tics action requires flake8 and pylint to be installed on the runner
-          # https://github.com/tiobe/tics-github-action/issues/410
-          python -m pip install flake8 pylint
           # pin tox to the current major version to avoid
           # workflows breaking all at once when a new major version is released.
           python -m pip install 'tox<5'
 
-      # The snap is built already in the check workflow and available as an artifact,
-      # but we want this TICS workflow to be independent,
-      # so we can run the workflow periodically or on demand.
+        # The snap is built already in the check workflow and available as an artifact,
+        # but we want this TICS workflow to be independent,
+        # so we can run the workflow periodically or on demand.
       - name: Build the snap
         id: build
         uses: canonical/action-build@v1
 
       - name: Test with tox and produce coverage report
-        run: tox -e tics
+        run: tox --workdir /tmp/tox -e tics
```

Comment:
**question:** 

Can you leave a comment here to explain why we need this change?